opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/ExpressionFilterScriptTest.java

```diff
@@ -109,14 +113,43 @@ void can_execute_expression_with_float_field() {
   }
 
   @Test
-  void can_execute_expression_with_date_field() {
+  void can_execute_expression_with_timestamp_field() {
```

Comment:
Yes, changes were put in place because aggregation IT like [testMinTimeStampPushedDown](https://github.com/opensearch-project/sql/blob/a924d5a16bc0f24ef77fb4af5ac7e967cca84a43/integ-test/src/test/java/org/opensearch/sql/sql/AggregationIT.java#L449) were failing. Since they now pass, then this case should be covered.